### PR TITLE
README: drop mention of ancestry (etcd.io)

### PR DESCRIPTION
**Pull request by chalin**
Signed-off-by: Patrice Chalin <chalin@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,8 +42,6 @@ notaryproject.dev is licensed under an [Apache 2.0 license](./LICENSE).
 
 The Notary Project documentation (e.g., `.md` files in the `/content/docs` folder) is licensed under a [CC-BY-4.0 license](./LICENSE).
 
-The notaryproject.dev site uses [Docsy](https://www.docsy.dev/), and was cloned from the [etcd-io/website](https://github.com/etcd-io/website/) repo to capture CNCF specific edits.
```
Comment (chalin) [inline README.md:45]: We already mention Docsy elsewhere in this page, and I don't feel that the rest of the sentence is relevant info to NP website contributors.

**Comment by nate-double-u** [comment]
> Is there a better solution? An addon, or?

I did find this: https://github.com/scottrigby/dco-gh-ui, but a browser extension isn't a great solution for general DCO use (especially if we're asking folks to click the 'edit this page' button and make quick changes through the UI)

**Comment by chalin** [comment]
Signed-off-by: Patrice Chalin <pchalin@gmail.com>

**Comment by chalin** [comment]
Thanks for the feedback @nate-double-u. Is there an easy way to fix this now that the PR has been created? Adding a comment to the PR doesn't do the trick.

**Comment by nate-double-u** [comment]
> Thanks for the feedback @nate-double-u. Is there an easy way to fix this now that the PR has been created? Adding a comment to the PR doesn't do the trick.

I don't think there is a way to do this using the GitHub UI 😞 -- [GitHub suggests](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message) using the `git commit --amend` command, suggesting that for these kinds of changes the terminal is required.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Pull request merged**